### ISSUE TITLE
Icon resize 

### ISSUE THREAD

**Issue by lilsherm09** [PrismLauncher/PrismLauncher#1812]
### Role

Main page, mod configuration and adding mods

### Suggestion

Having a setting that lets you change the modpacks icon, size of the mod icons and text in modlist, when finding mods to add being able to change the size of mod icons and text

### Benefit

QoL for some users and adds more customization for your preferences 

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

Basically just letting you change how big the mods/modpack icons are. Especially for people who don’t use alot of modpacks it be nice to have the icons bigger than a bunch of useless empty space. Maybe making it a setting you can change the size of it
